src/rar2fs.c

```diff
@@ -4688,43 +4688,85 @@ static int64_t get_blkdev_size(struct stat *st)
 #endif
 }
 
+#ifndef __CYGWIN__
+/*!
+ *****************************************************************************
+ *
+ ****************************************************************************/
+static int parse_fuse_fd(const char *mountpoint)
+{
+        int fd = -1;
+        unsigned int len = 0;
+
+        if (sscanf(mountpoint, "/dev/fd/%u%n", &fd, &len) == 1 && len == strlen(mountpoint)) {
```

Comment:
- We should try to adhere to the 80 char line width, so lets split the line right before checking len and indent+align properly.
- For short/simple one line conditionals like this, remove the superfluous { and }.